### PR TITLE
MODUL-323 RTE custom cleaning

### PR DESCRIPTION
**Pull request by gabra20**
- [x] Provide a small description of the changes introduced by this PR
the headers (h1, h2, ...) are now replaced by paragraphs.
- [x] Include links to issues
https://jira.dti.ulaval.ca/browse/MODUL-323
- [ ] Openshift deployment requested

### CODE REVIEW

File: src/utils/clean/htmlClean.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+export function filterByTag(tag: string, html: string): string {
+    const regex: RegExp = new RegExp(`<${tag}([^>]*)>([^]*)</${tag}>`, 'gmi');
+
+    return html.replace(regex, '');
+}
+
+export function filterByTags(tags: string[], html: string): string {
+    tags.forEach((tag) => { html = filterByTag(tag, html); });
```
Comment (jfnadeau) [inline src/utils/clean/htmlClean.ts]: Une regex dynamique comme createMultiTagsRegexString serait pas mal plus performant :-)

File: src/utils/clean/htmlClean.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+function createMultiTagsRegexString(tags: string[]): string {
+    // create a string for a multi tags regexp
+    let fullTagForRegex: string = '(';
+    for (let i: number = 0; i < tags.length; i++) {
```
Comment (jfnadeau) [inline src/utils/clean/htmlClean.ts]: tu pourrais remplacer par un tags.join('|')

File: src/utils/clean/htmlClean.ts (hunk 1 above)
Comment (gabra20) [reply src/utils/clean/htmlClean.ts]: oops, oublié d'utiliser createMultiTagsRegexString...

**Comment by jfnadeau** [comment]
conflit a regler

**Comment by gabra20** [comment]
plus de conflits.

**Pull request merged**